atd-toolbox/fatalities_backfill/fatalities_backfill.sql

```diff
@@ -0,0 +1,11 @@
+-- This is a one time script to populate the fatalities table with historical data
+
+INSERT INTO fatalities (crash_id, primaryperson_id)
+SELECT crash_id, primaryperson_id
+FROM atd_txdot_primaryperson
+WHERE prsn_injry_sev_id = 4;
+
+INSERT INTO fatalities (crash_id, person_id)
+SELECT crash_id, person_id
+FROM atd_txdot_person
+WHERE prsn_injry_sev_id = 4;
```

Comment:
Super not important, but if add a new line at the end of the file, GitHub won't flag it as missing and it will be posix compliant.

Reply:
fixed :)